docs/corpora_schema.md

```diff
@@ -34,32 +35,46 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
+*   AnnData requires that matrix layers MUST have the same dimension, so raw count matrices MUST include the same cells and genes as the final. Because it is impractical to
+    retain all barcodes in raw and final matrices, cells MUST be filtered from both. By contrast, those wishing to reuse datasets require access to raw gene expression values,
+    so genes MUST be present in both datasets. Summarizing, any cell barcodes that are removed from the data MUST be filtered from both raw and final matrices.
+    By contrast, genes MUST NOT be filtered from the raw matrix. Any genes that publishers wish to filter from the final matrix MAY have their values replaced by np.nan,
+    which will mask them from exploration.
+
+In addition to these general requirements, the following table describes the matrix data and layers requirements that are assay-specific. If cellxgene does not support
+an assay you would like to publish, please post an issue on this repository to start a conversation about extending the schema. If an entry in the table is empty,
+the cellxgene schema does not have any other requirements on data in those layers beyond the ones listed above. This is usually the case when there are many ways to
+produce the matrix layer in question.
+
+| Assay                                 | Raw requirements                                                                                                                              | Raw Required? | Final Requirements                                                                                 | Final Required?      | Other Layers |
+|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------|----------------------|--------------|
+| scRNA-seq (UMI, e.g. 10x v3)          | Values MUST be de-duplicated molecule counts.                                                                                                 | REQUIRED      |                                                                                                    | STRONGLY RECOMMENDED | OPTIONAL     |
+| scRNA-seq (non-UMI, e.g. SS2)         | Values MUST be one of read counts (e.g. FeatureCounts) or  estimated fragments (e.g. output of RSEM).                                         | REQUIRED      |                                                                                                    | STRONGLY RECOMMENDED | OPTIONAL     |
+| Accessibility (e.g. ATAC-seq, mC-seq) |                                                                                                                                               | NOT REQUIRED  | Values MUST correspond to gene features (e.g. HGNC symbols if human)                               | REQUIRED             | OPTIONAL     |
```

Comment:
If are removing the raw matrix requirement for ATAC-seq, [this](https://github.com/chanzuckerberg/single-cell-curation/issues/27) should be resolved ASAP. 

> One of `layer_description` must be "raw" (not "Raw", "Raw counts", etc). Otherwise there [will be an error](https://github.com/chanzuckerberg/corpora-data-portal/blob/ffca067b9e4aea237fa2bd7c7a9cbc5813ebd449/backend/corpora/dataset_processing/process.py#L257-L258) when processing the data in the corpora data portal.

Reply:
It was our decision that while there may be other forms of data suitable for recording "raw" accessibility data, that we cannot specify a form for raw (observations, genes) data. I agree that #27 should be resolved soon.